curator-client/pom.xml

```diff
@@ -62,12 +62,6 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
I am positive to this. I prefer real classes than mocks in most cases if not all. I do believe mock could provide "clean" tests, but the "clean" part could lost effect in refactor comparing to stupid/verbose/explicit hooks. Also, I am fearing about tests full of mocks. Removing this will force us to think about testability. That is good for me.